sdk/go/pulumi/rpc_test.go

```diff
@@ -467,7 +639,8 @@ func TestMarshalRoundtripNestedSecret(t *testing.T) {
 	assert.Nil(t, err)
 
 	if assert.Nil(t, err) {
-		assert.Equal(t, reflect.TypeOf(inputs).NumField(), len(resolved))
+		const resourceFields = 10
+		assert.Equal(t, reflect.TypeOf(inputs).NumField()-resourceFields, len(resolved))
```

Comment:
This isn't entirely obvious looking at the code. Could you add a comment as to why there are 10 less resolved fields?

Reply:
Certainly.